### DISCUSSION TITLE
Packet Loss vs. Distance - Showcase

### DISCUSSION THREAD

**Discussion by rhornig** [inet-framework/inet#589] [Showcases]
Discussion about the Packet Loss vs. Distance showcase

**Comment by priteshranjan01** [comment]
The URL pointed by "Source files location: `inet/showcases/wireless/errorrate`" is https://github.com/inet-framework/inet-showcases/tree/master/visualizer/earth 
I guess the correct URL should be https://github.com/inet-framework/inet-showcases/tree/master/wireless/errorrate

**Comment by adamgeorge309** [comment]
Fixed now. Thank you for pointing this out!

**Comment by Kakashi944** [comment]
I am quite new to Inet and Omnet. But my graph "PER vs distance" and "PER vs SNIR" don't have any result. I don't know why, I have run many simulations but still nothing showed up
